src/briefcase/integrations/linuxdeploy.py

```diff
@@ -48,6 +49,21 @@ def download_url(self) -> str:
     def file_path(self) -> Path:
         """The folder on the local filesystem that contains the file_name."""
 
+    @classmethod
+    def arch(cls, host_os: str, host_arch: str) -> str:
+        # always use the x86-64 arch on macOS since the
+        # manylinux image will always be x86-64 for macOS
+        arch = "x86_64" if host_os == "Darwin" else host_arch
```

Comment:
It's true in this implementation since this same code is used to determine the x86-64 manylinux image should always be used on macOS.

Nonetheless, this review has included a lot of questions about building AppImages on macOS. My intention was to support building x86_64 AppImages by using the x86_64 image of manylinux on macOS irrespective of hardware. Therefore, while the aarch64 image of manylinux works on Apple Silicon, you can't then build an x86_64 AppImage with that image....and without aarch64 from linuxdeploy, you won't be able to build any AppImage.

Does that help clarify our difference of position? I'm not sure exactly where we aren't aligned.

Reply:
Yes, I think that clarifies things. I think I was reading some of this PR as an attempt at "make all the architectures work everywhere"; keeping it to "native architecture only" definitely make sense as a scoping exercise, and avoids a bunch of gnarly compatibility issues.

That said - as a longer term issue, I'd be interested in whether running x86 docker on M1 Mac images is viable for Linux System and Windows - mostly because it would make my personal testing life a *lot* simpler.

Reply:
> I'd be interested in whether running x86 docker on M1 Mac images is viable for Linux System and Windows - mostly because it would make my personal testing life a _lot_ simpler.

That's is definitely in scope currently for #1392 😃